modules/sdk-core/src/bitgo/keychain/keychains.ts

```diff
@@ -264,6 +264,9 @@ export class Keychains implements IKeychains {
       // if the provider is undefined, we generate a local key and add the source details
       const key = this.create();
       _.extend(params, key);
+      if (params.passphrase !== undefined) {
+        _.extend(params, { encryptedPrv: this.bitgo.encrypt({ input: key.prv, password: params.passphrase }) });
+      }
```

Comment:
checking line 273, do we want to return the prv and the encryptedPrv ?

Reply:
No, the params here get sent to WP via line 272. 

line 273 annotates the response from WP with the prv, so the PRV doesn't actually get sent to WP.